docs/integrations/unity-catalog-spark.md

```diff
@@ -229,7 +229,7 @@ Let’s query the first five rows of the `marksheet` table.
 === "PySpark"
 
     ```python
-    sql("SELECT * FROM default.marksheet LIMIT5;").show()
+    sql("SELECT * FROM default.marksheet LIMIT 5").show()
```

Comment:
Thanks for fixing the spacing, could you add a semi-colon to the end of the statement?  Thanks!

Reply:
Added.